dns/opnblock/src/opnsense/mvc/app/models/OPNsense/opnblock/opnblock.xml

```diff
@@ -0,0 +1,23 @@
+<model>
+    <mount>//OPNsense/opnblock</mount>
+    <version>0.1</version>
+    <description>Ad-blocker for Unbound DNS</description>
+    <items>
+        <!-- container -->
+        <general>
+            <!-- fields -->
+            <Enabled type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </Enabled>
```

Comment:
keep all naming consistent - downcase to enabled so it matches the rest of your model